scripts/appserver_build.sh

```diff
@@ -46,7 +46,7 @@ chmod -w /app/drupal8/web/sites/default
 sed -i -e "s|name=\"SIMPLETEST_BASE_URL\" value=\"\"|name=\"SIMPLETEST_BASE_URL\" value=\"http:\/\/${LANDO_APP_NAME}.${LANDO_DOMAIN}\"|g" /app/config/phpunit.lando.xml
 if [ -f "/app/config/phpunit.lando.xml" ]; then
   echo "Copying PHPUnit config to Drupal webroot"
-  ln -sf /app/config/phpunit.lando.xml /app/drupal8/web/core/phpunit.xml
+  cp /app/config/phpunit.lando.xml /app/drupal8/web/core/phpunit.xml
```

Comment:
Does linking not work? Just thinking we will run into issues if people update the config/ version

Reply:
It does, but I've found copying the file into the container at runtime or when you need it to change keeps it consistent with other work I've done lately. Looking at it after a sleep or two I sense there's some variation in how things are generated/passed in whether that's a literal value in a file + the entire file copied in (like this example), a symlink or `sed` to rewrite certain parts of a file then copy it in.

My gut feeling is drawing me to literal + copy in, with sed + copy in where we need a dynamic value (eg: a Lando hostname). We can't achieve the latter with a symlink unless we expect people to either keep the exact same Lando hostnames or other config, or require them to exercise discipline to not commit any host-specific changes to the repo (never works :))